File: api/omp/codegen/lib/ompGetWtick/ompGetWtick.c
#include "ompGetWtick.h"
#include "omp.h"

double ompGetWtick(void)
{
  return omp_get_wtick();
}

void ompGetWtick_initialize(void)
{
}

void ompGetWtick_terminate(void)
{
}

File: api/omp4m/ompGetWtick.m
function tick = ompGetWtick
%Returns a double-precision floating point value equal 
%   to the number of seconds between successive clock ticks.
%
%    tick = ompGetWtick
% 
% See also ompGetWtime

%#codegen -args {}

tick = 0;
if ~strcmp(coder.target,'rtw'); return; end

coder.cinclude('omp.h');
coder.inline('always');
coder.allowpcode('plain')

tick = coder.ceval('omp_get_wtick');
